src/store/middleware/model-poller.test.ts

```diff
@@ -22,6 +21,10 @@ type Conn = {
   };
 };
 
+type Juju = {
```

Comment:
This typing could be imported from jujulib once the PRs are merged.

Reply:
Yeah good point, I've added a `TODO`.